content/md/connectivity/manage-your-connections.md

```diff
@@ -76,28 +99,96 @@ That being said, even in this case, we advise you to choose between the two opti
 
 # Grab your credentials
 
-Whenever you create a connection, the PIM automatically generates a set of credentials for you.
+Whenever you create a connection, the PIM automatically generates a set of credentials for you. These credentials are necessary if you want to make any API calls to the PIM. Don't hesitate to [learn more about the authentication over the API in our dedicated API website](https://api.akeneo.com/documentation/authentication.html).
 
 These credentials consist of 4 different strings:
 - the `client id`,
 - the `secret`,
 - the connection `username`,
 - the connection `password`.
 
-To access the client id, the secret and the username, go to `System` / `Connecitons`, click on the connection for which you want to see the credentials. They are displayed on the right side of the screen in the `Credentials` column.
+To access the client id, the secret and the username, go to `System` / `Connections`, click on the connection for which you want to see the credentials. They are displayed on the right side of the screen in the `Credentials` column.
 
-![Client id & secret in the connection form](client_id_and_secret_in_the_connection_form.png) <!-- TODO once all edit form cards are merged -->
+![Credentials in the connection form](../img/credentials-in-connection-form.png)
 
 You can easily copy/paste the credentials by clicking on the button next to them.
 
-![Copy/paste credentials](copy_paste_credentials.png) <!-- TODO once all edit form cards are merged -->
+![Copy/paste credentials](../img/copy-paste-credentials.png)
 
 The password is only showed once to you, just after the connection creation. So, make sure you save it somewhere.
 
-![Password is showed only once](password_showed_only_once.png) <!-- TODO once all edit form cards are merged -->
+![Password is showed only once](password-showed-only-once.png) <!-- TODO -->
+
+::: tips
+You forgot your password? Don't worry, you can regenerate a new one. Take a look at the [Regenerate your connection password](#regenerate-your-connection-password) section.
+:::
+
+Use these credentials to authenticate the API calls in the connector that matches your connection. For this step, take a look at this [guide](https://api.akeneo.com/getting-started/connect-the-pim-4x/step-3.html).
+
+# Revoke/regenerate your connection secret
+
+In some cases, your connection secret may have leaked and fallen into the wrong hands. So you want to revoke this secret, to avoid these "wrong hands" to access your PIM via the API.
```

Comment:
```suggestion
In some cases, your connection secret may have leaked and fallen into the wrong hands. So you want to revoke this secret, to avoid enabling these "wrong hands" to access your PIM via the API.
```